tests/e2e/e2e_exec_test.go

```diff
@@ -809,6 +809,36 @@ func (s *IntegrationTestSuite) executeValidatorBond(c *chain, valIdx int, valOpe
 }
 */
 
+func (s *IntegrationTestSuite) executeTokenizeSharesFailure(c *chain, valIdx int, amount, valOperAddress,
+	delegatorAddr, home, delegateFees string,
+) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	s.T().Logf("Executing gaiad tx liquid tokenize-share %s", c.id)
+
+	gaiaCommand := []string{
+		gaiadBinary,
+		txCommand,
+		liquidtypes.ModuleName,
+		"tokenize-share",
+		valOperAddress,
+		amount,
+		delegatorAddr,
+		fmt.Sprintf("--%s=%s", flags.FlagFrom, delegatorAddr),
+		fmt.Sprintf("--%s=%s", flags.FlagChainID, c.id),
+		fmt.Sprintf("--%s=%s", flags.FlagGasPrices, delegateFees),
+		fmt.Sprintf("--%s=%d", flags.FlagGas, 1000000),
+		"--keyring-backend=test",
+		fmt.Sprintf("--%s=%s", flags.FlagHome, home),
+		"--output=json",
+		"-y",
+	}
+
+	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, s.expectErrExecValidation(c, valIdx, true))
+	s.T().Logf("%s expected failure on execution of tokenize share tx from %s", delegatorAddr, valOperAddress)
```

Comment:
Feels kinda weird to have the same logic duplicated for a failure case. Maybe we can deduplicate and refactor out the success messages from every execute message. Imo they should be agnostic to whether they succeed or fail

Reply:
Yeah I agree, it's not super clean. Let me take a look at simplifying this when I merge in the e2e test refactors from main.